src/troute-network/setup.py

```diff
@@ -112,7 +112,7 @@ def build_extensions(self):
     sources=[
              "troute/network/reservoirs/rfc/rfc.{}".format(ext),
              ],
-    include_dirs=[np.get_include()],
+    include_dirs=[np.get_include(),  "troute/network/"],
```

Comment:
Is this an issue if you try to install from a different directory than the dir that has `setup.py`?

Reply:
I'm not sure about that, haha.